### PR TITLE
Add NestJS backend

### LINKED ISSUE #4: Initialize Backend

### PR DESCRIPTION
**Pull request by sid-707**
Closes #4

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,70 @@
+{
+  "name": "nest-typescript-starter",
+  "private": true,
+  "version": "1.0.0",
+  "description": "Nest TypeScript starter repository",
+  "license": "MIT",
+  "scripts": {
```
Comment (IndiW) [inline package.json]: thoughts on using pnpm over npm (I'm assuming you're using npm here)

**Review by IndiW — COMMENTED** [review]
should we put all this nestjs code in a backend folder? then we can put our frontend code in a /frontend folder

File: package.json (hunk 1 above)
Comment (sid-707) [reply package.json]: [Done](https://github.com/sid-707/BugCatcher/pull/6/commits/f9f434272200df733793f4566bd1bb6ccde054a7)

**Comment by sid-707** [comment]
> should we put all this nestjs code in a backend folder? then we can put our frontend code in a /frontend folder

[Done](https://github.com/sid-707/BugCatcher/pull/6/commits/0a98bf21cd741dfd9fa762454a290e18146bab0d)

**Review by IndiW — APPROVED** [review]

**Pull request merged**